js/README.md

```diff
@@ -1,111 +1,121 @@
+💡 First, verify you have the relevant [requirements](#Setup) 😉
 
-```console
-## Optional if not modifying any .proto file
-## Generate files using protobuf
-$ make generate
+Build and serve mobile UI:
 
-## Build and serve mobile UI
+```bash
 $ make metro.start
+```
 
-## Optional if using only one device
-## Run this with different ports to test with multiple devices
-$ BERTY_DAEMON_PORT=1337 make daemon.start
-$ BERTY_DAEMON_PORT=1338 make daemon.start
+Run iOS app in debug mode:
+
+```bash
+# Generate dependencies (Xcode files, gomobile, etc.):
+$ make ios.app_deps
 
-## iOS
-## Run iOS app in debug mode
 $ make ios.debug
-## Optional if using only one device
-## Run this with different iOS device names
-$ IOS_DEVICE=__IOS_DEVICE_NAME_1__ make ios.debug
-$ IOS_DEVICE=__IOS_DEVICE_NAME_2__ make ios.debug
+# or setting the device id:
+$ IOS_DEVICE="your-emulator-id" make ios.debug
+
+# 💡 You can check available virtual iOS devices with
+$ xcrun simctl list
+```
+
+Run Android in debug mode:
 
-💡 You can check available virtual iOS devices with `xcrun simctl list`
+```bash
+# Generate dependencies (Android files, gomobile, etc.):
+$ make android.app_deps
 
-## Android
-## Run Android app in debug mode
 $ make android.debug
-## Optional if using only one device
-## Run this with different Android device IDs
-$ ANDROID_DEVICE=__ANDROID_DEVICE_ID_1__ make android.debug
-$ ANDROID_DEVICE=__ANDROID_DEVICE_ID_2__ make android.debug
+# Optional if using only one device
+# Run this with different Android device IDs
+$ ANDROID_DEVICE="your-emulator-id" make android.debug
 
-💡 You can check available Android Virtual Devices with `adb devices`
+# 💡 You can check available Android Virtual Devices with
+$ adb devices
+```
+
+```bash
+# Optional if not modifying any .proto file
+# Generate files using protobuf
+$ make generate
+
+# Optional if using only one device
+# Run this with different ports to test with multiple devices
+$ BERTY_DAEMON_PORT=1337 make daemon.start
+$ BERTY_DAEMON_PORT=1338 make daemon.start
 
-## Web
-## Run Berty daemon for the web app
+# Web
+# Run Berty daemon for the web app
 make web.daemon.start
-## Run web app in debug mode
+# Run web app in debug mode
 $ make web.debug
-## Open http://localhost:3002/#/ip4/127.0.0.1/tcp/9092/grpcws in your browser
+# Open http://localhost:3002/#/ip4/127.0.0.1/tcp/9092/grpcws in your browser
+```
+
+List of all available commands:
+
+```bash
+$ make help
 ```
 
-## 🧳 Requirements
+## Setup
 
-### Requirements for working on iOS and Android apps
+### The Berty Messenger installation involves:
 
-- The [General React Native requirements](#general-react-native-requirements)
-- [iOS dev reqs](#ios-dev-requirements) **and/or** [Android dev reqs](#android-dev-requirements)
-- [Watchman](https://facebook.github.io/watchman/docs/install/) to enable live reloading
-- [Docker Desktop](https://docs.docker.com/docker-for-mac/install/)
+1. Installing the global dependencies
```

Comment:
"global dependencies" should be "common dependencies" because that term is used below.